app/extension/maven-plugin/pom.xml

```diff
@@ -17,19 +17,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>io.syndesis</groupId>
-    <artifactId>syndesis-rest-parent</artifactId>
+    <groupId>io.syndesis.extension</groupId>
+    <artifactId>extension-parent</artifactId>
     <version>1.2-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
-  <artifactId>syndesis-maven-plugin</artifactId>
-  <name>REST :: Maven Plugin</name>
+  <artifactId>extension-maven-plugin</artifactId>
```

Comment:
I wouldn't say this doesn't affect tooling...it effectively breaks tooling ;)

Reply:
Our code freeze is tomorrow and you introduce such a disruptive change at this point? Why was it required to change the Maven Coordinates of the plugin?

Reply:
Yes, that renaming may cause issues :sweat_smile:. Are projects generated via archetypes by tooling? do you also invoke the plugin directly?

Reply:
the example is completely duplicated in Fuse tooling codebase and embedded in built product.

we don't invoke the plugin directly

Reply:
How to proceed:

I need to know if these are now the final coordinates for the plugin 
and 
I need to know if there is already a staging repo hosting this plugin so I can test the tooling with the new plugin.

As already mentioned the code freeze is tomorrow. Every such kind of change u do after this code freeze has the potential of breaking the tooling's Syndesis Extension functionality. This will be lasting until we do a 10.3 release mid of April leaving possible users with a broken tooling in the time between. 

I just want to raise the level of awareness that changes in this project can have effects on the tooling. If you are unsure about side effects then please feel free to put tooling team as reviewers or as mentions. We will then help to figure out the risk.

Reply:
> we don't invoke the plugin directly

We invoke this plugin during the build of the project. So if the coords change then the plugin is not found. (or if we are lucky some old version is found instead)

Reply:
>> we don't invoke the plugin directly
> 
> We invoke this plugin during the build of the project. So if the coords change then the plugin is not found. (or if we are lucky some old version is found instead)

we don't rely on normal mvn build integration of m2e? We have a specific menu?

Reply:
We invoke the m2e build via Eclipse to create the deployable artifact.

Reply:
Extensions have now a broader scope so it is required to have such changes to keep the code-base clean and have better module naming. This change is also required to have a dedicated bom for extension so you do not need to specify spring-boot, camel and syndesis version.

As interim solution we can wrap the new plugin with the old one so the coordinates are still valid but I think the issue here is that tooling release cycle is not in sync with syndesis - or vice versa :) - so I think PM should discuss this, @rhuss what do you think ?

Reply:
> I think the issue here is that tooling release cycle is not in sync with syndesis - or vice versa :) - so I think PM should discuss this, @rhuss what do you think ?

it was decided by PM a year ago to decouple the release cycles of Fuse Tooling to be part of Jboss Tools.
For this specific release, we also took care that the Tech preview 3 is matching  the Fuse Tooling 10.2.0. Seems Tech preview 3 has shifted.

Reply:
> Extensions have now a broader scope so it is required to have such changes to keep the code-base clean and have better module naming. This change is also required to have a dedicated bom for extension so you do not need to specify spring-boot, camel and syndesis version.

Required for Tech preview 3 or for Tech preview 4?
Using a bom instead of the version requires to change also the Wizard UI on Fuse Tooling side.

> As interim solution we can wrap the new plugin with the old one so the coordinates are still valid

If for a tech preview 3, it might be a solution or to shift this modification to Tech Preview 4

Reply:
Well, this is part of the epic "Custom connectors" (#803) which targets TP4 so yeah, this targets TP4. TP3 branch has been cut yesterday afaik.

Reply:
Yes, 1.2.x is the TP3 branch.

Reply:
@lhein @apupier as this targets TP4 and TP3 has branched is this PR still a blocker ?

Reply:
The next stable Fuse Tooling release (10.3.0) - in normal process - is in 3 months (I mean after the 10.2.0 which has code freeze tomorrow). But there are Milestones publicly available every 3 weeks.
The Fuse Tooling will still work with Tech Preview 3 even after this modification.
With the PR as-is, it means that a dedicated release of Fuse Tooling would need to be planned (never happened yet) to have support of new plugin for Tech Preview 4 Technical Extension in a "stable" Fuse Tooling release.
As it is support for some Tech previews, I think it is fine to have it only in milestones but PM can decide otherwise.